src/pudl/metadata/fields.py

```diff
@@ -2805,8 +3474,10 @@
         "type": "boolean",
         "description": "Was an uprate or derate completed on this generator during the reporting year?",
     },
-    "utc_datetime": {"type": "datetime"},
-    "utility_attn": {"type": "string"},
+    "utc_datetime": {
+        "type": "datetime",
+        "description": ("Date and time converted to Coordinated Universal Time (UTC)."),
+    },
```

Comment:
I think there's another field similar to this called `operating_datetime_utc`, maybe we should investigate how similar they are. Also, I'm blanking but is there a reason we don't just convert all dates to utc? Or do we? 

This reminds me that there is definitely room to do a more thorough review of columns and potential column duplication, but it's out of scope right now.